SonarQube.Client/Api/V7_20/GetExclusionsRequest.cs

```diff
@@ -0,0 +1,88 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using SonarQube.Client.Models;
+using SonarQube.Client.Requests;
+
+namespace SonarQube.Client.Api.V7_20
+{
+    public class GetExclusionsRequest : RequestBase<ServerExclusions>, IGetExclusionsRequest
+    {
+        internal const string ExclusionsKey = "sonar.exclusions";
+        internal const string GlobalExclusionsKey = "sonar.global.exclusions";
+        internal const string InclusionsKey = "sonar.inclusions";
+        internal const string GlobalInclusionsKey = "sonar.global.inclusions";
+
+        [JsonProperty("component")]
+        public virtual string ProjectKey { get; set; }
+
+        [JsonProperty("keys")]
+        public virtual string Keys { get; } =
+            string.Join(",", ExclusionsKey, GlobalExclusionsKey, InclusionsKey, GlobalInclusionsKey);
```

Comment:
Minor: wouldn't just having a single hard-coded string be easier to read? I'm not sure the individual constants add anything except clutter.